src/css/custom.css

```diff
@@ -2,8 +2,22 @@
   --ifm-light-background-color: var(--ifm-color-emphasis-100);
 }
 
+html[data-theme='light'] {
+  --ifm-color-primary: #1554B7;
+  --ifm-link-color: #1554B7;
+
+  .DocSearch-Button-Placeholder {
+    color: #494B5F;
+  }
+}
+
 html[data-theme='dark'] {
-  --ifm-link-color: #679df5;
+  --ifm-color-primary: #7FA8F1;
+  --ifm-link-color: #7FA8F1;
+
+  .DocSearch-Button-Placeholder {
+    color: white;
```

Comment:
I think this can be set using the `--docsearch-muted-color` css variable.
This makes it easier to maintain when we upgrade docusaurus.

Reply:
Addressed in latest commit